src/collections/programs/sca-contributhon/index.mdx

```diff
@@ -1,7 +1,9 @@
 ---
-title: "She Code Africa Contributhon"
+title: "She Code Africa Contributhon 2020"
```

Comment:
This will be 2021, update the below frontmatter also.

Reply:
Done